apps/mull-ui/src/constants.ts

```diff
@@ -14,7 +14,7 @@ export const ROUTES = {
   MESSAGES: '/messages',
   GROUPCHAT: { url: '/messages/group-chat', displayName: 'Group Chat' },
   ANNOUNCEMENTS: { url: '/messages/announcements', displayName: 'Announcements' },
-  PROFILE: { DISPLAY: '/profile', EDIT: '/profile/edit' },
+  PROFILE: { DISPLAY: '/profile', EDIT: '/profile/edit', ADDFRIENDS: '/profile/add-friends' },
```

Comment:
A small nitpick, if you don't have to push any code other than this comment then feel free to merge lol
```suggestion
  PROFILE: { DISPLAY: '/profile', EDIT: '/profile/edit', ADD_FRIENDS: '/profile/add-friends' },
```